### PR TITLE
fix: :bug: incorrect logo url with slash at the end of `url_prefix`

### LINKED ISSUE #398: the logo disappeared
Ciao!

How are you?
Just the logo disappeared? How come!

image

### PR DESCRIPTION
**Pull request by Meeeeow**
**Type:** bug

The following has been addressed in the PR:
- fix: :bug: incorrect logo url with slash at the end of `url_prefix`

Resolves #398

### CODE REVIEW

File: src/utils/string.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+module.exports.spliceURL = function spliceURL() {
+  return Array.from(arguments).reduce((lastResult, current) => lastResult + current).replace(/([^:])(\/)+(.)/g, `$1/$3`);
```
Comment (juanpicado) [inline src/utils/string.js]: Why use arguments instead normal parameters?

Comment (Meeeeow) [reply src/utils/string.js]: Unlimited number of arguments, `spliceURL('https://example.org', '/future/', 'use')`

Comment (juanpicado) [reply src/utils/string.js]: I see, Now I remember you wanted to use spread operator.

Comment (juanpicado) [reply src/utils/string.js]: Could you rebase ?

Comment (Meeeeow) [reply src/utils/string.js]: Done

**Comment by juanpicado** [comment]
Shipped on `v.2.6.6`

**Pull request merged**